src/preferences/constants.h

```diff
@@ -17,4 +17,12 @@ enum class ScreenSaverPreference {
     PREVENT_ON_PLAY = 2
 };
 
+enum class MultiSamplingMode {
+    DISABLED = 0,
+    TWO = 2,
+    FOUR = 4,
+    EIGHT = 8,
+    SIXTEEN = 16
```

Comment:
enum _class_ members are usually not all uppercase, thats a legacy style choice still remaining from the "unscopped" non-class enums.

```suggestion
    Disabled = 0,
    Two = 2,
    Four = 4,
    Eight = 8,
    Sixteen = 16
```